CONTRIBUTING.md

```diff
@@ -118,30 +118,49 @@ Once you are satisfied with your PR you should ask someone to review it. Before
+3. Add the date of release in the `Changelog.md` in [ISO format](https://xkcd.com/1179/).
+4. After pushing these changes, create a pull request from `release/vX.Y.Z` towards `master` and merge it into `master`.
+
+### The actual release
+1. Open a working python3 virtual environment and make sure you have the latest versions of setuptools and wheel installed:
+`python3 -m pip install --upgrade setuptools wheel twine`
+2. Delete `build/` and `dist/` directories (from previous release) to avoid errors.
+3. Make sure you are in `master` and the latest commit you want to release and create the package:
+`python3 setup.py sdist bdist_wheel`
+4. Check the package with twine: `python3 -m twine check dist/*` If errors occur, fix them before the release or postpone.
+5. Create a [release tag](https://github.com/rl-institut/multi-vector-simulator/releases) on github.
+Please choose `master` as target and use `vX.Y.Z` as tag version. In the description field simply copy-paste the content of the `CHANGELOG`descriptions for this release and you're done!
+For help look into the [github release description](https://help.github.com/en/github/administering-a-repository/creating-releases).
+6. Pull `master` to your local repository and check if the new tag is there. (If you do not see the tag type `git fetch`.)
+7. Test your package:
+    1. Test the upload on test.pypi.org
+        ` twine upload --repository testpypi dist/*`
+    2. Test the installation: `pip install -i https://test.pypi.org/simple/ multi-vector-simulator --no-deps`
+    3. Then open a python console and try
+        `>>>from multi_vector_simulator import cli`
+8. If everything works as expected you can now upload the package version to PyPI
+    1. Check the credentials of our pypi@rl-institut.de account on https://pypi.org.
+    2. Type `twine upload dist/*`
+    3. Enter `__token__` for username and your pypi token for password. 
+9. If your release candidate works well you can now do the actual release: start from step 2 in "Before the release" and remove `rcX` from [`src/multi-vector-simulator/version.py`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/version.py).
+If you notice errors in the uploaded package, fix them and bump up `rc1` to `rc2` until you don't see any more mistakes.
+
+### After the release
+
+1. Locally, merge `release/vX.Y.Z` into `dev` and push to the remote version of dev.
+--> The idea is to avoid creating a merge commit in `dev` (because `master` would otherwise have two merge commits for this release once you merge the next release).
+2. Set version for next release in [`src/multi-vector-simulator/version.py`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/src/multi_vector_simulator/version.py) of the `dev`: for example `0.5.1dev`
+3. Add the structure for a new `unreleased` version to the [`CHANGELOG.md`](https://github.com/rl-institut/multi-vector-simulator/blob/dev/CHANGELOG.md) in `dev`:
```

Comment:
Push directly to dev?

Reply:
This is one of the rare example where you want to push directly to dev after merging the release branch locally The idea is to avoid having merge commit of the same release twice in `master`: once in `master` when we release and at the next release (if we had a release commit in dev, this one will eventually end up into `master` at the next release)

Reply:
What do you mean by that? Adding a comment saying "push to dev"?

Reply:
> What do you mean by that? Adding a comment saying "push to dev" to make sure that is does not stay locally?